wolfssl/wolfcrypt/settings.h

```diff
@@ -3586,6 +3586,12 @@ extern void uITRON4_free(void *p) ;
     #endif
 #endif
 
+#if defined(HAVE_DSA) || !defined(NO_DSA)
```

Comment:
HAVE_DSA isn't really a thing... we don't use it.

```
./configure --enable-dsa --disable-sha && make
configure: error: please disable DSA if disabling SHA-1.
```

Would prefer simple the following:

```
#if !defined(NO_DSA) && !defined(NO_SHA)
    #error "Please disable DSA if disabling SHA-1"
#endif
```